code/modules/mob/living/brain/robotic_brain.dm

```diff
@@ -16,6 +16,8 @@
 	var/requires_master = TRUE
 	var/mob/living/carbon/human/imprinted_master = null
 	var/ejected_flavor_text = "circuit"
+	/// If this is a posibrain, which will reject attempting to put a new ghost in it, because this a real brain we care about, not a robobrain
+	var/is_posibrain = FALSE
```

Comment:
I'd consider calling this something else like "can_be_reinhabited" or something that's a bit clearer towards its purpose

Reply:
done